### PR TITLE
Address Issue #1026 ChoiceGroup: Support image alt text

### LINKED ISSUE #1026: ChoiceGroup: Support image alt text
**Component**: https://fabricreact.azurewebsites.net/fabric-react/master/#/examples/**choicegroup**

**Requirement**: https://learn.microsoft.com/rgcontents/CONT15249/scormcontent/class/ms/non-text-content/alt-text/decorative-redundant.html#non-text-content_images_decorative-redundant_foreground

ChoiceGroup control supports passing imageSrc. It doesn't allow overriding the image alt text. Expose imageAlt as optional property in IChoiceGroupOption interface. Also allow it to be passed as "" (Empty) explicitly to meet above a11y requirement.

### PR DESCRIPTION
**Pull request by oengusmacinog**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #1026
- [ ] Include a change request file using `$ npm run change`

#### Description of changes

Added imageAlt and selectedImageAlt props to ChoiceGroup. 

imageAlt falls back to empty ('') by default giving the decorative images an empty alt tag (alt='') if text is not set. 

#### Focus areas to test

Test the ChoiceGroup in general.

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.tsx [hunk 1]
```diff
@@ -187,7 +188,8 @@ export class ChoiceGroup extends BaseComponent<IChoiceGroupProps, IChoiceGroupSt
                   }) }
               >
                 <Image
-                  src={ option.selectedImageSrc }
+                  src={ option.imageSrc ? option.imageSrc : '' }
```
Comment (mikewheaton) [inline packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.tsx]: Wouldn't this already output an empty attribute if `imageSrc` isn't defined?

File: packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.types.ts [hunk 2]
```diff
@@ -77,11 +77,22 @@ export interface IChoiceGroupOption extends React.HTMLAttributes<HTMLElement | H
...
+
   /**
    * The src of image for choice field which is selected.
    */
   selectedImageSrc?: string;
 
+  /**
+   * The alt of image for choice field which is selected. Defaults to imageAlt if not set.
+   */
+  selectedImageAlt?: string;
```
Comment (battletoilet) [inline packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.types.ts]: I don't think there is an immediate need for this so let's remove it.

File: packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.tsx [hunk 3]
```diff
@@ -187,7 +188,8 @@ export class ChoiceGroup extends BaseComponent<IChoiceGroupProps, IChoiceGroupSt
                   }) }
               >
                 <Image
-                  src={ option.selectedImageSrc }
+                  src={ option.imageSrc }
+                  alt={ option.selectedImageAlt ? option.selectedImageAlt : '' }
```
Comment (battletoilet) [inline packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.tsx]: Let's put the alt here but change the value to `option.imageAlt`

**Comment by betrue-final-final** [comment]
I haven't been able to check out these changes. For some reason my localhost isn't loading, though it builds fine.

**Pull request merged**